src/js/stores/BaseStore.d.ts

```diff
@@ -0,0 +1,6 @@
+import { EventEmitter } from "events";
+
+export default class BaseStore extends EventEmitter {
+  addChangeListener: (eventName: string, callback: void) => void;
```

Comment:
I think the type `void` is wrong here (at least judging from the [docs](https://www.typescriptlang.org/docs/handbook/basic-types.html#void). I think you want to express something like 

```diff
-addChangeListener: (eventName: string, callback: void) => void;
+addChangeListener: (eventName: string, callback: () => void) => void;
```